text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+achieving linear horizontal scalability in the number of channels. That is, more resource can be added as the number of 
+channels increase. However, the system channel is an exception: decoupling application channel consenters sets as 
+described above will cause its number of members to increase, and hence its performance to decrease. Joining and 
+leaving channels without a system channel solves this problem.     
+   
+Moreover, in order to start a new orderer and have it join an existing channel (perform 'on-boarding'), a new orderer 
+starts by scanning the system channel in order to discover existing channels and their membership. As the number of 
+channels increase, this scan prolongs the process of adding a new orderer.  
+    
+## Operational problems
+There are several operational problems that affect large scale deployment and hinder efficient management of cloud 
+environments using automatic tooling.
+     
+Channel creation (using the system channel) utilizes information contained in the system channel - e.g. MSP information 
+of the organization - in order to apply it onto the newly created channel. This create the convoluted situation in 
+which an organization that wants to update its details in the consortium must coordinate that action with third parties 
+that have write privileges to the system channel.
+
+There is no good way to list all the channels that an orderer is a member of. 
+
+There is no good way to dispose of the resources associated with a channel, once an orderer is no longer a member of 
+the consenters set of a that given channel.
+
+In many cloud environments, it is necessary to restart an orderer in order for it to discover immediately that it has 
+been added to a channel it was not a part of previously, because otherwise detection takes minutes. 
+
+   
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+We introduce a new mode of channel participation management that does not depend on the existence of a system channel.
+
+We introduce the ability to start an OSN process without any channels. The OSN exposes a new **"channel participation" 
+API** that allows the operator to join and leave channels on that specific OSN.   
+
+A **"local orderer channel participation administrator"** is a role that is associated with the operation and 
+administration of an OSN. The identity associated with this role is able to invoke a new **"channel participation" 
+API**, and perform the following commands:
+
+- **Join** this OSN to a new or an existing channel,
+- **Remove** this OSN from an existing channel,
+- **List** the channels this OSN is a member of.   
+
+The new mode of channel participation management is meant to be incompatible with channel creation using a system 
+channel, in the sense that if the system channel exists, the `Join` operation will not be supported. 
```

Comment:
Does this have to be the case?  Seems like it should be possible to support both ... but I'll read about "migration" first

Reply:
"Mixed mode" is a problem... things can stop working in ugly ways, e.g. on-boarding. There are solutions of course, but they are complicated and costly. Therefore, for simplicity, I think we should avoid that and encourage users to use either one or the other. 

In addition, moving from system-channel mode to participation-api style is easy (delete system-channel on all OSNs). The reverse, going back to system-channel style once there are already app-channels  is complicated. Here too we choose simplicity and do not allow the reverse path. The only exception is bootstrap: starting from empty OSNs and creating the system channel.